packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -831,23 +913,30 @@ void main() {
         ),
       ),
     );
-
-    expect(tester.getSemantics(find.byKey(expandedKey)), matchesSemantics(
-      label: 'Expanded',
+    final Finder expandedIcon = find.descendant(
```

Comment:
How did the semantics tree change that expandedKey doesn't get us to the right semantics node anymore?

Reply:
previously the mergesemantics wrapped the entire header row, where expandedKey is one of its children.
Now the semantics only wrapped the icon in the header row. thus I have to grab the sibling of the header to get the correct semantics

Reply:
This still sounds like the tree changes in unexpected ways... Let's look at it together when you have a minute.